packages/proxmark3-git/build.sh

```diff
@@ -2,13 +2,15 @@ TERMUX_PKG_HOMEPAGE="https://github.com/RfidResearchGroup/proxmark3"
 TERMUX_PKG_DESCRIPTION="The Swiss Army Knife of RFID Research - RRG/Iceman repo"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@laro.se>"
-_COMMIT=1527677bc44a9a93e3997b8e362bc8b3cac6db7e
-TERMUX_PKG_VERSION="2022.11.22-${_COMMIT:0:8}"
-TERMUX_PKG_SHA256="97f9ab71074ba5bf9e016d31339ff1847eae4471488d80dab01da354bdd289f7"
+_COMMIT=9986cc68857a3f522dfa96e59faef78a47063ad2
+TERMUX_PKG_VERSION="2022.11.29-${_COMMIT:0:8}"
+TERMUX_PKG_SHA256="f97f25a558dd890468ad1f70cfac562b87e75f2765d51bf070a81d7df61ac31f"
 TERMUX_PKG_SRCURL="https://github.com/RfidResearchGroup/proxmark3/archive/${_COMMIT}.tar.gz"
 TERMUX_PKG_DEPENDS="libbz2, libc++, readline"
 TERMUX_PKG_BUILD_IN_SRC="true"
 TERMUX_PKG_BLACKLISTED_ARCHES="i686, x86_64"
+TERMUX_PKG_CONFLICTS="proxmark3"
+TERMUX_PKG_REVISION=1
```

Comment:
Reset revision when bumping version.

Reply:
Revision is typically used for releasing a new binary package. If a new binary package does not needed to release then revision does not need to bump.

Revision should always be reset (to `0`) when bumping version, as in this PR.

Reply:
> Revision is typically used for releasing a new binary package.

More precisely, for releasing a new binary package *with the same version as before*.